frontend/src/store/appStore.js

```diff
@@ -1,13 +1,10 @@
-import { createStore, applyMiddleware } from 'redux'
+import { createStore, applyMiddleware, compose } from 'redux'
 import thunk from 'redux-thunk'
 import recruitmentApp from './../reducers/actionReducers.js'
-
+import { composeWithDevTools } from 'redux-devtools-extension'
 export default function configureStore() {
   return createStore(
     recruitmentApp,
-    typeof window !== 'undefined' &&
-      window.__REDUX_DEVTOOLS_EXTENSION__ &&
-      window.__REDUX_DEVTOOLS_EXTENSION__(),
-    applyMiddleware(thunk)
+    /* preloadedState, */ composeWithDevTools(applyMiddleware(thunk))
```

Comment:
this file may have some merge conflicts with angad's pr... we'll see. nothing wrong about this though. just thought you would like to know that.

Reply:
^ yeah i had to do this composition thing too. I think your stuff should work with my version of this file since I did this and added other things so merging shouldn't be too difficult.

Reply:
Yeah the reason I changed it was before the preloadedState was being set to false so that was messing up things when the site first loaded.